controller/src/beerocks/bml/bml.cpp

```diff
@@ -564,8 +564,9 @@ int bml_set_dcs_continuous_scan_enable(BML_CTX ctx, const char *radio_mac, int e
         return (-BML_RET_INVALID_ARGS);
     }
 
-    // TODO: call suitable bml api
-    return BML_RET_OP_FAILED;
+    auto pBML = static_cast<bml_internal *>(ctx);
+    return pBML->set_dcs_continuous_scan_enable(
+        network_utils::mac_from_string(std::string(radio_mac)), enable);
 }
 
 int bml_get_dcs_continuous_scan_enable(BML_CTX ctx, const char *radio_mac, int *output_enable)
```

Comment:
Change to `int bml_get_dcs_continuous_scan_enable(BML_CTX ctx, const char *radio_mac, int *enable)`

Or even better as Vitaly suggested - why not use reference?

Reply:
In the BML I can't use a reference, since I'm not sure how well it will be imported in C
I can change the `bml_internal` to use a reference instead